testsuite/MDAnalysisTests/transformations/test_fit.py

```diff
@@ -99,8 +99,9 @@ def test_fit_translation_no_masses(fit_universe):
     # create a universe without masses
     ref_u  = make_Universe()
     # what happens Universe without masses is given?
-    with pytest.raises(AttributeError): 
+    with pytest.raises(ValueError) as exc:
```

Comment:
Changing this to `TypeError` as it is raised in 

https://github.com/MDAnalysis/mdanalysis/blob/5f513831301492ffcf054c7b57ee6092bf153c41/package/MDAnalysis/lib/util.py#L1355-L1361

should do the job. Sorry for oversseing this.

Reply:
If I make the changes to this then what will happen to the `assert` statement.